lib/galaxy/datatypes/hubAssembly.py

```diff
@@ -0,0 +1,59 @@
+"""
+HubAssembly datatype
+"""
+import logging
+import galaxy.version as version
+
+# Support for Galaxy <= 16.01
+if version.VERSION_MAJOR <= "16.01":
+    from galaxy.datatypes.images import Html
+else:
+    from galaxy.datatypes.text import Html
```

Comment:
Ok, I was afraid about the case where somebody wanted to use the datatype on 16.01 or earlier (and it was my case for various reasons). So as you wish :)